app/models/sitcom.rb

```diff
@@ -28,7 +28,11 @@ def end_year
     self[:end_year] || "Still Running"
   end
 
-  def owner?(current_user)
-    user == current_user
+  def owner_or_admin?(current_user)
```

Comment:
This is a good method refactor, however I won't accept this pull request until you write unit tests for this method.

There should be at least 3 separate specs in your user_spec.rb that test all the different possible cases for the 'owner_or_admin?' method returns.

For an example of how you can write these unit tests look here: https://github.com/LaunchAcademy/event_horizon/blob/master/spec/models/user_spec.rb on line 23-52 the unit tests covering the '#can_edit?' method are an example of what you want to write (test style/wise) for unit tests for this method